.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+# Changes
+<!-- describe your changes briefly -->
+
+* [ ] I also added the config to docserv-config
```

Comment:
Hm, I am not sure about the wording here. E.g. the `main` branches are usually not on docs.suse.com ... i.e.: there does not need to be a change in every case.

Not sure whether this proposal is better or worse:

```suggestion
* [ ] This change affects documentation already published on docs.suse.com as well
    * [ ] I made an MR to the `docserv-config` repo
    * [ ] Please update the `docserv-config` repo for me
```

Reply:
Yeah, it was a bad, quick hack before the next meeting. I take your suggestion.